regolith/helpers/u_milestonehelper.py

```diff
@@ -110,161 +102,173 @@ def construct_global_ctx(self):
                         "Please rerun your command adding '--type' "
-                        f"and typing a type from this list: {MILESTONE_TYPES}")
-                if rc.type:
-                    if rc.type in MILESTONE_TYPES:
-                        doc.update({'type': rc.type})
-                    else:
-                        raise ValueError(
-                            "ERROR: The type you have specified is not recognized. "
-                            "Please rerun your command adding '--type' "
-                            f"and giving a type from this list: {MILESTONE_TYPES}")
-                if rc.finish:
-                    rc.status = "finished"
-                    doc.update({'end_date': now})
-                    notes = doc.get("notes", [])
-                    notes_with_closed_items = [note.replace('()', '(x)', 1) for note in notes]
-                    doc["notes"] = notes_with_closed_items              
-                    if identifier == 'deliverable':
-                        name = 'deliverable'
-                    else:
-                        name = doc['name']
-                    print(
-                        "The milestone {} has been marked as finished in prum {}".format(
-                            name, key))
-                if rc.status:
-                    doc.update({'status': rc.status})
-                if rc.audience:
-                    doc.update({'audience': rc.audience})
-                if rc.due_date:
-                    doc.update({'due_date': rc.due_date})
-                doc['due_date'] = get_due_date(doc)
-                if rc.objective:
-                    doc.update({'objective': rc.objective})
-                if rc.notes:
-                    notes = doc.get("notes", [])
-                    notes.extend(rc.notes)
-                    doc["notes"] = notes
-                if identifier == 'milestones':
-                    if rc.name:
-                        doc.update({'name': rc.name})
-                    new_mil.append(doc)
+                        f"and giving a type from this list: {MILESTONE_TYPES}")
+            else:
+                mil.update({'type': 'meeting'})
+            mil.update({'identifier': "milestones"})
+            new_mil.append(mil)
+            pdoc = {'milestones': mil}
+            new_all = deepcopy(all_milestones)
+            for i, j in zip(index_list, new_all):
+                if j['identifier'] == 'milestones':
+                    new_mil.append(j)
+            for mile in new_mil:
+                del mile['identifier']
+            new_mil.sort(key=lambda x: x['due_date'], reverse=False)
+            pdoc.update({'milestones': new_mil})
+            rc.client.update_one(rc.database, rc.coll, {'_id': rc.projectum_id}, pdoc)
+            print("{} has been updated in projecta".format(rc.projectum_id))
+        else:
+            for uuid in rc.milestone_uuid:
+                pdoc = {}
```

Comment:
to keep the code a bit cleaner, when initalizing use
```
pdoc, upd_mil, all_miles, id = {}, [], [], []

Reply:
cleaned